CMakeLists.txt

```diff
@@ -56,7 +56,10 @@ add_subdirectory(apps)
 add_subdirectory(tests)
 
 set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "libavahi-client3, avahi-daemon")
+set(SERVUS_DEBIAN_PACKAGE_DEPENDS)
+if (avahi-client_FOUND)
+  list(APPEND SERVUS_DEBIAN_PACKAGE_DEPENDS "libavahi-client3, avahi-daemon")
```

Comment:
The syntax is now: NAME_PACKAGE_DEB_DEPENDS, and it can be a CMake list:
https://github.com/Eyescale/CMake/blob/master/CommonCPack.cmake#L17
so this line should be:
list(APPEND SERVUS_PACKAGE_DEB_DEPENDS libavahi-client3 avahi-daemon)